app/src/main/java/io/neurolab/fragments/MemoryGraphFragment.java

```diff
@@ -0,0 +1,195 @@
+package io.neurolab.fragments;
+
+import android.content.Intent;
+import android.graphics.Color;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v7.app.AppCompatActivity;
+import android.util.Log;
+import android.view.WindowManager;
+
+import com.github.mikephil.charting.charts.LineChart;
+import com.github.mikephil.charting.components.Legend;
+import com.github.mikephil.charting.components.XAxis;
+import com.github.mikephil.charting.components.YAxis;
+import com.github.mikephil.charting.data.Entry;
+import com.github.mikephil.charting.data.LineData;
+import com.github.mikephil.charting.data.LineDataSet;
+import com.github.mikephil.charting.highlight.Highlight;
+import com.github.mikephil.charting.interfaces.datasets.ILineDataSet;
+import com.github.mikephil.charting.listener.OnChartValueSelectedListener;
+import com.github.mikephil.charting.utils.ColorTemplate;
```

Comment:
remove the unused import

Reply:
Also, what are the axes labels and units of measurements? Would look good if you make it descriptive enough. Also, there is a scope for color scheme improvement as it doesn't fully resonate. @adityastic what do you suggest?

Reply:
Yup! I am open to color combination suggestions. And I will make the axes labels descriptive. Thanks.